release

```diff
@@ -315,19 +318,27 @@
         'git checkout master',
         `git pull --rebase ${origin} master --strategy=theirs`,
         `git checkout release/${newVersion} -- CHANGELOG.md`,
-        `node -e "var newVersion = '${newVersion}'; ${stringifyFunction(buildCommand)}"`,
+        `node -e "var newVersion = '${newVersion}'; ${stringifyFunction(updatePackageVersions)}"`,
         'git add CHANGELOG.md',
         'git add package.json',
         'git add tools/scripts/release/npm_assets/package.json',
         `git commit -m "update version number in package.json to ${newVersion}"`,
         `git push ${origin} master`
     );
 
-    function buildCommand () {
-      var json = require('./package.json');
-          json.version = newVersion;
-
-      fs.writeFileSync('./package.json', JSON.stringify(json, null, 2));
+    /**
+     * When updating the 'master' branch, update both the source repository 'package.json',
+     * and also the npm version that is used when deploying the releases to npm.
+     */
+    function updatePackageVersions () {
+      [
+        './package.json',
+        './tools/scripts/release/npm_assets/package.json'
+      ].forEach(filePath => {
```

Comment:
I consider that... except `updatePackageVersions` is actually stringified to be *evaluated* in a sub node process. I worried about try it...